src/Microsoft.TestPlatform.ObjectModel/TestCase.cs

```diff
@@ -234,10 +333,27 @@ private Guid GetTestId()
                 // do nothing
             }
 
-            string testcaseFullName = this.ExecutorUri + source + this.FullyQualifiedName;
+            var testcaseFullName = this.ExecutorUri + source; 
```

Comment:
Note that we still need to handle parameters in the case of a Theory or TestGroup of test cases that are only distinguished by parameters. If this is to be handled in a later PR, then it would be good to note it here.